hal/src/peripherals/usb/d5x/bus.rs

```diff
@@ -543,15 +583,38 @@ impl UsbBus {
             desc,
             buffers: RefCell::new(BufferAllocator::new()),
             endpoints: RefCell::new(AllEndpoints::new()),
+            pclk,
+            ahb_clk,
+            apb_clk,
         };
 
         Self {
             inner: Mutex::new(RefCell::new(inner)),
         }
     }
+
+    pub fn free(
```

Comment:
Should this api be renamed to `into_inner` to follow other rust api patterns. E.g. https://doc.rust-lang.org/stable/core/cell/struct.LazyCell.html#method.into_inner